src/cocoa/toga_cocoa/widgets/table.py

```diff
@@ -60,14 +60,14 @@ def create(self):
         # Add the layout constraints
         self.add_constraints()
 
-    def insert_row(self, node):
-        node._impl = TogaNodeData.alloc().init()
-        node._impl.node = node
-
-        self.node[node._impl] = node
-
-    def remove_row(self, node):
-        del self.node[node._impl]
+    # def insert_row(self, node):
+    #     node._impl = TogaNodeData.alloc().init()
+    #     node._impl.node = node
+    #
+    #     self.node[node._impl] = node
+    #
+    # def remove_row(self, node):
+    #     del self.node[node._impl]
```

Comment:
This can be deleted. Tables shouldn't have a native insert method; insertion is done to the data source.